Client/Frontend/Browser/ClipboardBarDisplayHandler.swift

```diff
@@ -87,7 +87,7 @@ class ClipboardBarDisplayHandler: NSObject, URLChangeDelegate {
         // Ugly hack to ensure we wait until we're finished restoring the session on the first tab
         // before checking if we should display the clipboard bar.
         guard sessionRestored,
-            !url.absoluteString.hasPrefix("\(WebServer.sharedInstance.base)/about/sessionrestore?history=") else {
+            !url.absoluteString.hasPrefix("\(InternalURL.baseUrl)/\(SessionRestoreHandler.path)?history=") else {
```

Comment:
Note: This change doesn't `fix` anything. It is just cleanup. The URLs are actually the exact same :)